src/OrchardCore.Themes/TheBlogTheme/Views/Content-BlogPost.Summary.liquid

```diff
@@ -1,5 +1,5 @@
-{% cache "blog-post-summary" %}
-{% cache Model.ContentItem.ContentItemId %}
+{% assign cacheId = "blog-post-summary-" | append: Model.ContentItem.ContentItemId %}
```

Comment:
Why both in the `id` and as a dependency?

Reply:
The dependency causes the cached block to be invalidated when the content changes.

The cache id gives us a unique cache item for each blog post.

Reply:
But you pass an id, not a content item to the dependency. How does it know it's a content item id?